src/dos/program_tree.cpp

```diff
@@ -191,9 +190,9 @@ void TREE::PreRender()
 		// If current code page misses one or more characters used to
 		// draw the tree, use standard 7-bit ASCII characters
 
-		std::string tmp_str;
-		to_dos("─├│└", tmp_str);
-		if (tmp_str.find('\0') != std::string::npos) {
+		const auto tmp = to_dos("─├│└");
+		// TODO[C++23]: Use 'contains()' instead of 'find()'
+		if (tmp.find('\0') != std::string::npos) {
 			use_ascii_fallback = true;
 		}
```

Comment:
Usage case: Also discarding the returned string.

Reply:
As above, changed the fallback mode.